inc/packages/did/class-web.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Get PLC Web document.
+ *
+ * @package FAIR
+ */
+
+namespace FAIR\Packages\DID;
+
+/**
+ * Class Web.
+ */
+class Web implements DID {
+	const TYPE = 'web';
+
+	/**
+	 * Decentralized ID.
+	 *
+	 * @var string
+	 */
+	protected string $id;
```

Comment:
I think we should rename this to `$did` as well as other `id` references (`$id`, `get_id()`) in the `DID` interface and its implementing classes (PLC and Web at the moment).

The DID as the `id` pertains to certain documents/properties per the spec, but a DID itself isn't its own ID; that's the method-specific-id. Since both DID and method-specific-id contain "id", rather than use `$id` at all in the DID interface and its implementing classes (PLC and Web), using `$did` makes it clear.

Reply:
I don't think we're using the MSID component for anything.

Reply:
Oh I know we're not storing the method-specific-id separately and we may never do so.

I meant that anyone coming in who knows about DID syntax will see a DID interface and implementing classes with properties for a method and an ID, and `id` is ambiguous - at first reading, it could be the DID or the method-specific-id to be appended after the method. It requires reading the docblock to confirm what `id` actually is, and my initial thought is that I don't think that should be necessary.

But since I don't know the reason why `id` was chosen over `did` for the property name, I'm only seeing part of the picture, so my question ultimately boils down to "`id` instead of `did` - how come?"

Reply:
I used `id` since I tend to use `id` universally as the property for something that is the primary key/unique identifier for a model. The DID string is the unique identifier for the `DID` object, just as post IDs are the unique identifier for posts.

> at first reading, it could be the DID or the method-specific-id to be appended after the method

I don't see this as confusing personally, but ymmv. The MSID is not the ID, in the same way that `/foo/bar` is not a full URI without the protocol and host attached. Calling the MSID the ID would be way more confusing, and as per other discussions I don't think we ever really want to be dealing with the separated MSID.